nutrition_ai/lib/screens/authenticated/settings.dart

```diff
@@ -140,9 +144,13 @@ class _SettingsScreenState extends State<SettingsScreen> {
               const SizedBox(height: 20),
               SelectInput(
                 controller: goalController,
-                defaultValue: user.goal.toString(),
+                defaultValue: (user.goal).isNotEmpty ? user.goal : 'none',
                 name: 'Goal',
                 items: const [
+                  DropdownMenuItem(
+                    value: 'none',
+                    child: Text('No Goal'),
+                  ),
```

Comment:
Is no goal is the same as maintain weight?

Reply:
ideally I would want to get rid of this and make goal a required field but for now it will be treated as maintain